lib/game/game_manager.ex

```diff
@@ -21,6 +21,22 @@ defmodule Reactor.GameManager do
     GenServer.call(@name, {:get_games})
   end
 
+  def get_users(game_id) do
+    game_id
+    |> to_reg
+    |> GenServer.call({:get_users})
+  end
+
+  def add_user_to_game(game_id, user) do
+    game_id
+    |> to_reg
+    |> GenServer.cast({:add_user, user})
```

Comment:
I do a cast to add users and then I get all the users later here:

https://github.com/elijahkim/reactor/pull/5/files#diff-1fb6039d5e4d42490e42443a5238f2f2R24

I'm not sure what best practice is